internal/collector/collector_test.go

```diff
@@ -151,3 +151,46 @@ func TestHwCollector(t *testing.T) {
 		t.Errorf("unexpected collecting result:\n%s", err)
 	}
 }
+
+func TestCrmCollector(t *testing.T) {
+	s := miniredis.RunT(t)
+
+	os.Setenv("REDIS_ADDRESS", s.Addr())
```

Comment:
:smile: I hope no one will decide to run multiple tests at once in the future. I suggest removing this and passing the config through some struct.

Reply:
Yeah, this might cause some issues. However, now Redis client is configured via ENV variables only.
This would need to be restructured in some other way.

https://github.com/vinted/sonic-exporter/blob/0c3d1466515eea5c1eac51ef82de4932c4c9bafc/pkg/redis/client.go#L31-L44

Reply:
@GiedriusS To mitigate the issue I implemented TestMain in [be369d9](https://github.com/vinted/sonic-exporter/pull/18/commits/be369d9ecbec7be185594da1948e8006feff3a74)

This should setup test env before all the tests and tear it down afterwards. We will not overwrite ENV variables inside Test suites this way.